src/nyc_trees/nyc_trees/context_processors.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.conf import settings
+
+
+def nyc_bounds(request):
+    return {'NYC_BOUNDS_XMIN': float(settings.NYC_BOUNDS[0]),
```

Comment:
I was having a lot of trouble unpacking the bounds tuple in the template, so I just destructure it here.